### ISSUE TITLE
 Rejailbreaking with Electra ends up in respringloop new TweakInjector update

### ISSUE THREAD

**Issue by dyarfaradj** [coolstar/electra1131#27]
I'm on iOS 11.2.1 Electra(latest) using both Sileo and Cydia. iPhone 7

So like 10-14 days ago there was like 20 updates on Cydia (some electra packages) I ended up updating all of them and everything seemed to be fine, however my phone randomly restarted after few days (never happened before and I'm a heavey jb user) so when I tried to rejailbreak using latest electra it got me into a respring loop. ONLY solutions is to disable all tweaks with iCleaner and then rejailbreak and then turn on all tweaks on iCleaner.

I bet it has to do with TweakInjector or something. Is it possible to contact coolstar? I've seen other have this issues lately too. I've never had any issues with electra until now with the rejailbreak shit. And sometimes my phone reboots too.. Haven't changed or installed new tweaks after updating these 20 packages so I don't think it has to do with any of my tweaks to do. Thanks

**Comment by viggou** [comment]
This seems like the same issue people are having on unc0ver with "reload system daemons"

**Comment by dyarfaradj** [comment]
@viggou yeah I noticed that but I haven’t touched unc0ver so i don’t know

**Comment by gummykage** [comment]
Having the exact same issue.

**Comment by dyarfaradj** [comment]
@gummykage I still have the same issue, do you still have the same issue or have you found a way to avoid this?

**Comment by dyarfaradj** [comment]
No updates on this issue? Can you post older .deb of TweakInjector so I can avoid this issue please!

**Comment by viggou** [comment]
Are you on Electra1131 or 1141? I'm using 1141 with at least 100 tweaks and don't have the issue.

**Comment by dyarfaradj** [comment]
I'm using 1141 the latest, had no issues with it til the TweakInjector on Cydia as I mentioned in OP

**Comment by viggou** [comment]
Weird... Idk I think your best bet is to restore rootfs with unc0ver and rejailbreak with Electra. It saves your repos and tweak settings, so you just need to reinstall your stuff.

**Comment by dyarfaradj** [comment]
> Weird... Idk I think your best bet is to restore rootfs with unc0ver and rejailbreak with Electra. It saves your repos and tweak settings, so you just need to reinstall your stuff.

does it removes my phone data or only the jailbreak exceo repos and tweak data?

**Comment by viggou** [comment]
It only removes your tweaks and your jailbreak. Your tweak settings, repos and all your apps, photos, etc. is saved.

**Comment by dyarfaradj** [comment]
> It only removes your tweaks and your jailbreak. Your tweak settings, repos and all your apps, photos, etc. is saved.

Alright, thanks a lot for the information. Do you know any tweak that can backup and then restore tweaks? I don't wan to manually reinstall all of my tweaks. I knew a tweak in like iOS 9 but I couldn't find any for ios 11

**Comment by viggou** [comment]
> > It only removes your tweaks and your jailbreak. Your tweak settings, repos and all your apps, photos, etc. is saved.
> 
> Alright, thanks a lot for the information. Do you know any tweak that can backup and then restore tweaks? I don't wan to manually reinstall all of my tweaks. I knew a tweak in like iOS 9 but I couldn't find any for ios 11

Sorry I only know how to export a list of your tweaks. I think I say something called AZBackup or something but I don't remember.
